actors/account/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use num_traits::FromPrimitive;
 
 use fil_actors_runtime::builtin::singletons::SYSTEM_ACTOR_ADDR;
 use fil_actors_runtime::runtime::{ActorCode, Runtime};
-use fil_actors_runtime::{actor_dispatch, ActorDowncast};
+use fil_actors_runtime::{actor_dispatch, restrict_internal_api, ActorDowncast};
```

Comment:
why is `restrict_internal_api` needed for import when it is not used?

Reply:
It's used by the `actor_dispatch` macro at the end of the file.